Makefile

```diff
@@ -44,6 +44,14 @@ APPS2 := kmesh-cmd
 APPS3 := mdacore
 APPS4 := kmesh-cni
 
+TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
+	bpf/kmesh/bpf2go/kmeshcgroupsockworkload_bpfeb.go \
+	bpf/kmesh/bpf2go/kmeshcgroupsockworkload_bpfel.go \
```

Comment:
IC similar as other files. But i am thinking maybe we need add all of them back once we add unit test, otherwise `go test ` would fail.

But first let we remove them.